### PR TITLE
Fix background color default for empty states

### PR DESCRIPTION
**Pull request by vvalentin-lucca**
## Description

-----

-----

### CODE REVIEW

File: stories/documentation/feedback/empty-state/html&css/empty-state-page.stories.scss [hunk 1]
```diff
@@ -1,4 +1,3 @@
 :host {
 	display: block;
-	height: 100vh;
```
Comment (Supamiu) [inline stories/documentation/feedback/empty-state/html&css/empty-state-page.stories.scss:3]: Doesn't this make the story very small?

**Review by jeremie-lucca — APPROVED** [review]

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2515/storybook

File: stories/documentation/feedback/empty-state/html&css/empty-state-page.stories.scss (hunk 1 above)
Comment (vvalentin-lucca) [reply stories/documentation/feedback/empty-state/html&css/empty-state-page.stories.scss:3]: I've added a min-height in another PR to deal with this.

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/11894/ui-diff)

**Pull request merged**